core/templates/base-components/oppia-footer.component.ts

```diff
@@ -39,7 +39,8 @@ export class OppiaFooterComponent {
 
   SHORT_COMMIT_HASH = AppConstants.SHORT_COMMIT_HASH;
 
-  versionInformationIsShown: boolean = this.router.url === '/about';
+  versionInformationIsShown: boolean = (
+    this.router.url === '/about' && !AppConstants.DEV_MODE);
```

Comment:
I am curious about !DEV_MODE vs ProdMode. Don't we have a third mode as well now? According to the PR description I think we should directly check for Prod mode instead of inferring it from the Dev mode variable.